### ISSUE TITLE
Windows misplaced when restoring more than one windows

### ISSUE THREAD

**Issue by keianhzo** [MozillaReality/FirefoxReality#1728]
## Configuration

Firefox Reality version: 1.4
Firefox Reality build ID: 9a5f7c0

Hardware: All

## Steps to Reproduce

1. Open FxR
2. Open a second window
3. Click on the home button and quit the app
4. Launch FxR again
5. Move the windows

## Current Behavior

The windows are misplaced after moving them around.

![screencap](https://user-images.githubusercontent.com/837184/64100972-0f19f700-cd6d-11e9-9b53-4799db78a062.png)

## Expected Behavior

Windows should be correctly placed.

## Error Logs and Stack Traces

<details open>

```
```

</details>

**Comment by MortimerGoro** [comment]
The EnterVR button is not visible in the web, so that means GV didn't return VR displays or it resturned `vrDisplay.capabilities.canPresent` false. A reload fixes the issue.

I think it's the same issue as #966

**Comment by jvonitter** [comment]
Need some help reproducing this

**Comment by bluemarvin** [comment]
From the screen shot it looks like one of the windows is not curved. But have not been able to reproduce.

**Comment by keianhzo** [comment]
I'm having issues to reproduce this but have definitely seen it, will keep on trying and update with better STRs.

**Comment by keianhzo** [comment]
@jvonitter I can't find consistent STRs (although I've seen it happening a few times) so  wouldn't consider it a major issue for 1.4 unless @Softvision-GeluHaiduc can find reliable STRs.

**Comment by Softvision-GeluHaiduc** [comment]
I have tried to reproduce this issue with the latest Firefox Reality 1.4-32500011 (RC4) installed on the Oculus Go, Quest and HTC Vive Focus Plus headsets. 
I've managed to consistently reproduce this issue on the Oculus Go headset using these steps:

1. Go to the following website: "amazon.com".
2. Click the "+" new window button from the menu tray and go the following website: "youtube.com".
3. Press the oculus button on the controller and quit the app.
4. Reopen the app and move the window in position 2 (center window) to the left.
5. Observe the placement of the windows.

I was not able to reproduce this issue on the Oculus Quest and HTC Vive Focus Plus.
I've attached a screen recording of the issue [here](https://drive.google.com/file/d/1xno-bmm_tU-ETswGKORcr_AWnSg-9r_q/view?usp=sharing), and the logcat file [here](https://drive.google.com/file/d/1zIKcrzwId73Uf080PkT5qwdrWmsrr-r8/view?usp=sharing).
